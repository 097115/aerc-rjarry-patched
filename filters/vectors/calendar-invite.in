BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//hacksw/handcal//NONSGML v1.0//EN
CALSCALE:GREGORIAN
METHOD:PUBLISH
BEGIN:VEVENT
DESCRIPTION:A description.\nWith multiple lines.
DTEND:29991231T323000Z
DTSTAMP:19990101T000000Z
DTSTART:20000101T123000Z
LOCATION:Some Location
ORGANIZER:MAILTO:invalid@example.org
SUMMARY:Test Event
END:VEVENT
END:VCALENDAR
