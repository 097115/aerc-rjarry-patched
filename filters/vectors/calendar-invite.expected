
  This is a meeting PUBLISH

  SUMMARY       Test Event
  START         2000/01/01 12:30:00 
  END           2999/12/31 32:30:00 
  LOCATION      Some Location
  ORGANIZER     <invalid@example.org>

A description.
With multiple lines.
